internal/handlers/health.go

```diff
@@ -0,0 +1,16 @@
+package handlers
+
+import (
+	"net/http"
+
+	ihttp "github.com/Pigmice2733/peregrine-backend/internal/http"
```

Comment:
This isn't actually an issue per se, but every time I see `ihttp` I'm immediately going to think it's a typo. If only this wasn't a name collision... the only alternative package name I can think of is "rest" and I don't think that captures the intended purpose of the "ihttp" package. If you can think of a decent alternative we should use it, otherwise 🤷‍♂️

Reply:
I dislike it too but :man_shrugging: